test/AngularSpec.js

```diff
@@ -1,7 +1,16 @@
 'use strict';
 
+// Lots of typed array globals are used in this file and ESLint is
+// not smart enough to understand the `typeof !== 'undefined'` guards.
+/* eslint-env es6 */
```

Comment:
And it is considerably better that having it enabled for the whole `test/` directory :smiley: